src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -238,6 +239,162 @@
                 </DataTrigger>
             </Style.Triggers>
         </Style>
+
+        <!-- Flows Combo-box styles -->
```

Comment:
As I've said earlier, I don't know any other way to change the styles without overriding the control template and redeclaring all the elements in it. From all the search that I did, this is the way to do it, and that's why I've also done it in the past for the list view.

If you disagree and think that this is not the way, please explain how you've done it.

Reply:
If we're changing the structure of the control then redefining the template makes sense; if all we are going is changing the colours then it feels wrong, but I don't know of an alternative that works in VS.